engine/src/flutter/impeller/renderer/backend/vulkan/pipeline_library_vk.h

```diff
@@ -46,6 +46,7 @@ class PipelineLibraryVK final
   ComputePipelineMap compute_pipelines_ IPLR_GUARDED_BY(
       compute_pipelines_mutex_);
   std::atomic_size_t frames_acquired_ = 0u;
+  uint64_t pipeline_key_ = 0;
```

Comment:
I'd start counting at 1 instead of at zero to keep uninitialized data to mean it's invalid.